source/_integrations/mqtt.markdown

```diff
@@ -28,10 +28,6 @@ mqtt:
 
 You can also use the [embedded MQTT broker](/docs/mqtt/broker#embedded-broker). A separate broker is advised for more stability.
 
-<div class='note warning'>
-As of release 0.92, the embedded broker has been marked as deprecated. This means bugs may not be fixed, and the broker functionality will be removed in a future release.
```

Comment:
This feature has not been removed from the codebase, still true. We should not remove this, unless it has been removed from the codebase as well.